presto-main/src/main/java/com/facebook/presto/operator/project/PageProcessor.java

```diff
@@ -308,6 +301,17 @@ private ProcessBatchResult processBatch(int batchSize)
         }
     }
 
+    @VisibleForTesting
+    public List<PageProjection> getProjections()
+    {
+        return projections;
+    }
+
+    private static boolean isUnloadedLazyBlock(Block block)
```

Comment:
"unloaded" block sounds weird. Maybe we can use `isNotLoadedLazyBlock`, or you can inline this at its call site and add a comment.